### DISCUSSION TITLE
[docker] composer vs composer/composer

### DISCUSSION THREAD

**Discussion by rino1** [composer/composer#12004] [General]
https://hub.docker.com/_/composer/
https://hub.docker.com/r/composer/composer 

Difference between the two images?

**Comment by xabbuh** [comment]
> Non "official-images" build of Composer container, same as the "composer" one with faster releases

taken from https://hub.docker.com/r/composer/composer

**Reply by rino1** [reply]
Then you might as well use the unofficial image.
What is the advantage of using the official image if releases are slower?
I would like to understand...

**Reply by xabbuh** [reply]
~You may trust the Composer maintainers more than some arbitrary person publishing unofficial images for example.~

(this was wrong, see https://github.com/composer/composer/discussions/12004#discussioncomment-9660147)

**Reply by glaubinix** [reply]
Just to clarify, https://hub.docker.com/_/composer/ are official docker images by the Docker team. The Composer maintainers don't have any influence over those releases. As already mentioned, new release usually appear there slower. https://hub.docker.com/r/composer/composer are releases by the Composer maintainers. I believe the second one was created because official releases usually take a while.

**Reply by xabbuh** [reply]
Oh, thanks for the clarification @glaubinix. I stand corrected then.

**Reply by rino1** [reply]
OK,  but what changes on a technical level?
In addition to the fast versions, why should I choose composer rather than composer/composer?
Why are there two images of the same package?
What does it mean?

**Reply by Seldaek** [reply]
Because someone decided to publish it as official package, which is great because it's got the official docker stamp, and it's short, but then we realized this was sometimes a bit slow to publish so we started publishing ourselves for those who want faster releases.
